### ISSUE TITLE
toolcfg not saved

### ISSUE THREAD

**Issue by AndiLeni** [eclipse-sumo/sumo#14506]
Hello,

I tried saving the toolcfg of the randomTrips tool in netedit.
However, no file is created and if I create the file manually and select it as the target file it stays empty.

![image](https://github.com/eclipse-sumo/sumo/assets/13385771/aea3a477-d9d3-45ee-b44c-4e6f66ebc08c)

**SUMO-version:**
1.19.0

**operating system:**
Windows 10

**Comment by namdre** [comment]
I was able to replicate and fix a problem with boolean options. If it still doesn't work with the latest dev version, please report a minimal set of options to replicate the issue.

**Comment by AndiLeni** [comment]
@namdre I tried the latest github build but unfortunately it is still not saving anything.
It does not seem to matter which options I set in the gui. The toolcfg file is not created.

These are the settings I applied: 
![image](https://github.com/eclipse-sumo/sumo/assets/13385771/c6a029bf-b52b-49ab-820c-12aafd31fb9b)

Could this be caused by whitespaces in the path?

netedit however reports "Konfiguration für randomTrips gespeichert."
